backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -187,25 +188,23 @@ def _get_sender(self, user: User=None) -> str:
     @staticmethod
     def log_notification(
             config: CFG,
+            msg: str,
```

Comment:
adding `msg` may be interpreted as "msg is the email message" (because there are also `recipient` and `subject` parameters). I suggest to keep msg and to rename r`ecipient` and `subject` to `mail_recipient` and `mail_subject`

Reply:
done, i've use **email** instead of mail for better code coherency and  also change log_notification (in EmailManager) to **log_email_notification** in  to be more explicit.